docs/central-intro.rst

```diff
@@ -33,7 +33,6 @@ Here are some of the major features we support today:
    - Connectable to analysis and dashboard applications like Excel, Power BI, or R over OData
    - Server-managed datasets for registration/follow-up workflows
  - Integrated checklist-based help system
- - Optional encrypted off-site data backups to Google Drive
```

Comment:
We also include this line in ["What's coming in Central"](https://forum.getodk.org/t/whats-coming-in-central/19677) in the forum. @yanokwa, would you up for removing that line?